### PR TITLE
Design uplift for the generated project

### PR DESCRIPTION
**Pull request by StanevPrime**
#### Overview

image

image

image

image

image

#### Checklist

*Implementation*
- [ ] Implementation matches ticket acceptance criteria and technical notes
- [ ] Manually tested against Acceptance Criteria
- [ ] UI checked in Light / Dark mode

*Stability*
- [ ] Checked if changes affect any features and verified affected features work as expected
- [ ] Added unit tests for new code and verified existing tests work as expected

*Code quality*
- [x] Updated `CHANGELOG.md`, `README.md` and package versions in `pubspec.yaml`
- [x] If an unrelated part of the codebase needs to be updated or refactored, create tickets with proposed changes

### CODE REVIEW

File: packages/rx_bloc_cli/mason_templates/bricks/feature_profile/__brick__/lib/feature_profile/views/profile_page.dart [hunk 1]
```diff
@@ -33,212 +33,205 @@ class ProfilePage extends StatefulWidget {
 }
 
 class _ProfilePageState extends State<ProfilePage> {
-  @override
-  void initState() {
-     context.read<ProfileBlocType>().events.loadNotificationsSettings(); {{#enable_pin_code}}
```
Comment (StanevPrime) [inline packages/rx_bloc_cli/mason_templates/bricks/feature_profile/__brick__/lib/feature_profile/views/profile_page.dart:38]: @JvnSlv looking at the BLoC it seems that we have added an initial trigger `startWith(null)`, which probably makes the invocation of the `loadNotificationsSettings()` redundant. Please confirm this change won't have negative impact.

**Review by GoranPrime — APPROVED** [review]
The code looks good, however on the Android emulator at least I feel like the name & profile pic are too close together:

image

I'm approving anyway to not block the PR

**Comment by StanevPrime** [comment]
> The code looks good, however on the Android emulator at least I feel like the name & profile pic are too close together:
> 
> image
> I'm approving anyway to not block the PR

@JvnSlv could you please address this issue in a separate PR ?

File: packages/rx_bloc_cli/mason_templates/bricks/feature_profile/__brick__/lib/feature_profile/views/profile_page.dart (hunk 1 above)
Comment (JvnSlv) [reply packages/rx_bloc_cli/mason_templates/bricks/feature_profile/__brick__/lib/feature_profile/views/profile_page.dart:38]: You're right; `startWith(null)` makes `loadNotificationsSettings()` redundant, and removing it won't have any negative impact.

**Pull request merged**